eng/pipelines/aggregate-reports.yml

```diff
@@ -68,32 +68,10 @@ stages:
       - job: ComplianceTools
         timeoutInMinutes: 120
         steps:
-          - pwsh: |
-              azcopy copy "https://azuresdkartifacts.blob.core.windows.net/policheck/JavaScriptPoliCheckExclusion.mdb?$(azuresdk-policheck-blob-SAS)" `
-              "$(Build.BinariesDirectory)"
-            displayName: 'Download PoliCheck Exclusion Database'
-            condition: succeededOrFailed()
-
-          - task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@2
-            displayName: 'Run PoliCheck'
-            inputs:
-              targetType: F
-              targetArgument: '$(Build.SourcesDirectory)'
-              result: PoliCheck.sarif
-              optionsFC: 0
-              optionsXS: 1
-              optionsPE: 1|2|3|4
-              optionsRulesDBPath: "$(Build.BinariesDirectory)/JavaScriptPoliCheckExclusion.mdb"
-              optionsUEPATH: "$(Build.SourcesDirectory)/eng/guardian-tools/policheck/PolicheckExclusions.xml"
-            condition: succeededOrFailed()
-
-          - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
-            displayName: 'Post Analysis (PoliCheck)'
-            inputs:
-              GdnBreakAllTools: false
-              GdnBreakGdnToolPoliCheck: true
-              GdnBreakGdnToolPoliCheckSeverity: Warning
-            continueOnError: true
+          - template: /eng/common/pipelines/templates/steps/policheck.yml
```

Comment:
The error you are hitting is that you cannot download the exclusion database in the public pipelines because we cannot access any secrets. We may need to think about making those publicly available.

Reply:
You are right, but can that be addressed by adding the SAS secret to the pipeline.
I have cretaed this https://github.com/Azure/azure-sdk-tools/pull/2041 , if am thinking correctly after merging that and runing the pipeline-generation pipeline those secrets will propagate across.

Reply:
We cannot access secrets at all in the public team which is by design.

Reply:
Ok, i will make the exclusiong publicly available.